doc/documentation_data_format.md

```diff
@@ -283,7 +220,90 @@ numeric value or `inf` (lower-case) for steady-state measurements.
 - `replicateId` [STRING, OPTIONAL]
 
   The replicateId can be used to discern replicates with the same
-  datasetId, which is helpful for plotting e.g. error bars.
+  `datasetId`, which is helpful for plotting e.g. error bars.
+
+
+### Observables table
+
+Parameter estimation requires linking experimental observations to the model
+of interest. Therefore, one needs to define observables (model outputs) and
+respective noise models, which represent the measurement process.
+Since parameter estimation is beyond the scope of SBML, there exists no
+standard way to specify observables (model outputs) and respective noise
+models. Therefore, in PEtab observables are specified in a separate table
+as described in the following. This allows for a clear separation of the
+observation model and the underlying dynamic model, which allows, in  most
+cases, to reuse any existing SBML model without modifications.
+
+The observable table has the following columns:
+
+| observableId | observableName | observableFormula | [observableTransformation] | noiseFormula | [noiseDistribution] |
```

Comment:
possibly needs edit if https://github.com/ICB-DCM/PEtab/issues/242 is implemented

Reply:
yes, but decision pending.